src/preset_cli/cli/superset/sync/dbt/datasets.py

```diff
@@ -403,14 +436,15 @@ def sync_datasets(  # pylint: disable=too-many-locals, too-many-arguments
             continue
 
         # update column metadata
-        if dbt_columns := model.get("columns"):
+        if (dbt_columns := model.get("columns")) or calculated_columns:
```

Comment:
Super nit, but I find this hard to read. I would split it into 2 lines:

```suggestion
        dbt_columns = model.get("columns")
        if dbt_columns or calculated_columns:
```

Reply:
sure! gonna change that too